constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintLayout.java

```diff
@@ -1247,8 +1246,7 @@ private void setChildrenConstraints() {
         }
     }
 
-
-    protected void applyConstraintsFromLayoutParams(boolean isInEditMode,
+protected void applyConstraintsFromLayoutParams(boolean isInEditMode,
```

Comment:
Same here on whitespace
```suggestion
    protected void applyConstraintsFromLayoutParams(boolean isInEditMode,
```